### ISSUE TITLE
Support suitable Lirc drivers ("plugins") for capturing and sending

### ISSUE THREAD

**Issue by bengtmartensson** [bengtmartensson/harctoolboxbundle#25]
For plugin Lirc drivers which are not of the "LIRCCODE" type (see http://lirc.org/html/driver-api.html), support should be possible. There is already the possibility to run them through the "mode2" program, but this has severe shortcomings.

Possible implementation: A daemon program load a lirc plugin and communicates with IrScrutinizer, preferably using a Girs (http://harctoolbox.org/Girs.html) over, e.g. sockets. It should run without root, provided that the driver allows it. (Even if it runs as root, thus the root-ness will be isolated as compared with the mode2-solution.)

Preferably both sending and receiving. Linux only, at least the first phase.

**Comment by bengtmartensson** [comment]
Actually, this may not be a very smart idea. There are really not that many "sane" Lirc (not LIRCCODE) drivers, sorting out  the ones that is not based on hardware misuse (audio) or non-hardware (file), or disguised GPIOs (FTDI), leaves commandir, iguanaIR, irtoy, uirt2_raw and usb_uirt_raw (the two latter different driver for the same device, USB/UIRT). Out of these, IrToy is already supported. CommandIR appears to be out of business, USB/UIRT fairly rare, and sort-of expensive. Remains  Iguana, which would be interesting to support.

Bottom line: better to support Iguana directly.
